lib/ClimaCoreTempestRemap/test/netcdf.jl

```diff
@@ -153,6 +154,118 @@ end
     end
 end
 
+function test_warp(coords)
+    λ, ϕ = coords.long, coords.lat
+    FT = eltype(λ)
+    ϕₘ = FT(0) # degrees (equator)
+    λₘ = FT(3 / 2 * 180)  # degrees
+    rₘ = @. FT(acos(sind(ϕₘ) * sind(ϕ) + cosd(ϕₘ) * cosd(ϕ) * cosd(λ - λₘ))) # Great circle distance (rads)
+    Rₘ = FT(3π / 4) # Moutain radius
+    ζₘ = FT(π / 16) # Mountain oscillation half-width
+    h₀ = FT(10)
+    if rₘ < Rₘ
+        zₛ = FT(h₀ / 2) * (1 + cospi(rₘ / Rₘ))
+    else
+        zₛ = @. FT(0)
```

Comment:
Does this ever get executed? Looks like it should maybe be `zₛ = FT(0)`?

Reply:
Both are used (which one depends on the coords).